frontend/javascripts/test/fixtures/hybridtracing_object.ts

```diff
@@ -0,0 +1,103 @@
+import update from "immutability-helper";
+import mockRequire from "mock-require";
+import type { SkeletonTracing } from "oxalis/store";
+import { initialState as defaultState } from "test/fixtures/volumetracing_object";
+
+import DiffableMap from "libs/diffable_map";
+import EdgeCollection from "oxalis/model/edge_collection";
+import { MISSING_GROUP_ID } from "oxalis/view/right-border-tabs/trees_tab/tree_hierarchy_view_helpers";
+import { TreeTypeEnum } from "oxalis/constants";
+import type { APIColorLayer } from "types/api_flow_types";
+
+mockRequire("app", {
+  currentUser: {
+    firstName: "SCM",
+    lastName: "Boy",
+  },
+});
```

Comment:
I quickly dug into the rabbit hole and I think this hasn't been necessary since 2017 :sweat_smile: could you remove it here and everywhere else in the repo and check that everything still works?

Reply:
still works :D